addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -178,4 +162,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
```

Comment:
Should they (both) be advanced?

Reply:
Hm, maybe yes. I looked into other bindings and they declare similar channel-types as not advanced. I will change that.